source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+
+.. My vote is to merge the manifest with the sample metadata
+
+Let's start by downloading the manifest and corresponding sequences.
+
+.. download::
+   :url: https://data.qiime2.org/2019.7/tutorials/pd-mice/manifest
+   :saveas: manifest
+
+.. download::
+   :url: https://data.qiime2.org/2019.7/tutorials/pd-mice/demultiplexed_seqs.zip
+   :saveas: demuliplexed_seqs.zip
+
+You'll need to unzip the directory of sequences.
+
+.. command-block::
+
+   unzip demuliplexed_seqs.zip
+
+You can use the ``head`` command to check the first five lines of the sample manifest.
+
+.. command-block::
+   :no-exec:
+
+   head -n 6 manifest
+
+When using the single end manifest format, a sample name can only appear in one line and can only map to one sequencing file. In a paired end manifest, the sample name should appear twice. The **absolute-filepath** for each sample must be an `absolute path`_, which specifies the “full” location of the file. We do that here using the ``$PWD`` variable, which uses the local absolute directory. The **direction** column can only have two values: *forward* and *reverse*.
+
+We’ll use the manifest to import our data.
+
+.. command-block::
+
+   qiime tools import \
+     --type "SampleData[SequencesWithQuality]" \
+     --input-format SingleEndFastqManifestPhred33V2 \
+     --input-path ./manifest \
+     --output-path ./demux_seqs.qza
+
+Let’s check the sequences and the sequencing depth of the samples using the ``qiime demux summarize`` command. It provides information about the number of sequences in each sample, as well as the quality of the sequences.
+
+Before running the command, let’s review the help documentation to make sure we understand the arguments.
+
+.. command-block::
+   :no-exec:
+
+   qiime demux summarize --help
+
+Based on the documentation, we should pass the demultiplexed sequences that we imported as the ``--i-data`` argument, since this takes a ``SequencesWithQuality]`` semantic type, and that’s the type of data we imported. We’ll specify the location we want the visualization by passing the output path to ``--o-visualization``. However, to speed up the command process, we’ll change the ``--p-n`` parameter to 1000. This means that rather than resampling the sequences 10000 times (the default number) to get the quality score, we’ll only re-sample them 1000 times.
```

Comment:
```suggestion
Based on the documentation, we should pass the demultiplexed sequences that we imported as the ``--i-data`` argument, since this takes a ``SampleData[SequencesWithQuality`` semantic type, and that’s the type of data we just imported. We’ll specify the location we want to save the visualization to by specifying the output path to ``--o-visualization``. However, to speed up the command process, we’ll change the ``--p-n`` parameter to 1000. This means that rather than resampling the sequences 10000 times (the default number) to get the quality score, we’ll only re-sample them 1000 times.
```

Not sure about changing the default to 1000, seems like the difference in runtime is pretty negligible. What do you think, @jwdebelius?